v03_pipeline/lib/tasks/write_new_variants_table.py

```diff
@@ -265,7 +265,7 @@ def create_table(self) -> hl.Table:
                 ar_ht = ar_ht.union(ar_ht_chunk)
             new_variants_ht = new_variants_ht.join(ar_ht, 'left')
 
-        return new_variants_ht.annotate_globals(
+        return new_variants_ht.select_globals(
```

Comment:
I noticed stuff hanging around... they get cleared off the next step anyways, but this is cleaner!